packages/widgetbook_exception/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: widgetbook_exception
+description: A dart exception handler for Widgetbook
+version: 0.0.1-beta.1
+homepage: https://www.widgetbook.io?utm_source=pub.dev&utm_medium=link&utm_campaign=widgetbook_exception
+repository: https://github.com/widgetbook/widgetbook
+issue_tracker: https://github.com/widgetbook/widgetbook/issues
+documentation: https://docs.widgetbook.io/widgetbook-exception/overview
```

Comment:
For the documentation I would link to the github page containing this package so they can read the readme there and have easy access to the code of the package.

Reply:
for the documentation i have changed it to
`https://github.com/widgetbook/widgetbook/tree/main/packages/widgetbook-exception#readme`
which will be the link to the readme once the package is ready